proto/data.proto

```diff
@@ -252,6 +252,22 @@ message Transaction {
   optional NodeList certain_nodes = 12 [(gogoproto.nullable) = false];
 }
 
+// Lease contains information about leader leases including the
+// expiration and lease holder.
+message Lease {
+  // The expiration is in unix nanos and is set according to the wall
+  // clock at the lease requestor / grantee.
+  optional int64 expiration = 1 [(gogoproto.nullable) = false];
+  // The leadership term for this lease.
+  optional uint64 term = 2 [(gogoproto.nullable) = false];
+  // The node ID of replica holding the lease.
```

Comment:
is there a specific reason to send the store_id and node_id along? I would prefer just sending the RaftID (which the range knows), and being less redundant.